lib/n3h-mod-connection-wss/wss-connection.js

```diff
@@ -233,6 +233,11 @@ class ConnectionBackendWss extends AsyncClass {
         return
       }
 
+      if (e.code === 'ECONNREFUSED') {
```

Comment:
a connection dropping is not really an error, so much as it will be a normal occurence

Reply:
:+1: